main.tf

```diff
@@ -95,6 +97,16 @@ resource "google_compute_url_map" "urlmap" {
     redirect_response_code = "MOVED_PERMANENTLY_DEFAULT"
     strip_query            = false
   }
+
+  default_custom_error_response_policy {
+    error_response_rule {
+      match_response_codes   = local.default_error_response_rule["match_response_codes"]
+      path                   = local.default_error_response_rule["path"]
+      override_response_code = local.default_error_response_rule["override_response_code"]
+    }
+    error_service = var.default_custom_error_response_policy["error_service"]
+  }
```

Comment:
Isn't there 1 lb for all the services in Greenroom? I think we might actually need to move this to the `default_custom_error_response_policy` under `path_matchers` so 404 errors for `dashboard-api` doesn't get a 200.

`https://greenroom-api.brandlive-dev.com/asdfjfjfjf` is returning 200 and the `index.html`